static/get_grpcurl

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
```

Comment:
LICENSE?

Reply:
The other files in this repo don't appear to have a license file.